### PR TITLE
(1/2) Ante test for BAYC max supply

### PR DESCRIPTION
**Pull request by crypto-thawra**
# Purpose
- There is a function `reserveApes` in the [BAYC contract code ](https://etherscan.io/address/0xbc4ca0eda7647a8ab7c2061c2e118a18a936f13d#code) that can allow Yuga Labs to mint additional apes, over the 10k advertised limit. 
- this test is to assert the invariant that there will never be more than 10,000 bored apes.
- to add the first non de-fi test in the annte-community-test repository

# Testing
- boilerplate unit test
- see https://github.com/antefinance/ante-community-tests/pull/100 for more

### Update
As soon as I opened, I googled this known issue and Yuga labs revoked their access today (June 8)
- [news story](https://www.theblockcrypto.com/post/150764/yuga-labs-revokes-code-that-allowed-creation-of-infinite-bored-apes)
- [trasnfer ownership transaction](https://etherscan.io/tx/0x2c903f1d6bf8a0d07a16c947a752b3c6e9338411ac372ea0444445f7d0281e6c#eventlog)
Still useful test to have

### CODE REVIEW

**Comment by delalunia** [comment]
While this is WIP, this is a great check on Yuga Labs!! I mean, even seeing how many would stake vs challenge this test is a signal for how trustworthy people think Yuga Labs are!

By the way, have you joined our discord yet?

**Comment by crypto-thawra** [comment]
Does the ante core team want this deployed _before_ requesting a review and having this merged?

**Comment by delalunia** [comment]
> Does the ante core team want this deployed _before_ requesting a review and having this merged?

Nope! You don't need to have it deployed before having it merged! I'll be reviewing it and adding it in xD

**Review by delalunia — APPROVED** [review]

**Pull request merged**
